packages/flame/lib/src/components/shape_component.dart

```diff
@@ -27,9 +35,70 @@ class ShapeComponent extends PositionComponent {
   @override
   void render(Canvas canvas) {
     super.render(canvas);
-    shape.render(canvas, shapePaint);
+    shape.render(canvas, paint);
   }
 
   @override
   bool containsPoint(Vector2 point) => shape.containsPoint(point);
 }
+
+class CircleComponent extends ShapeComponent {
+  CircleComponent(
+    double radius, {
+    Vector2? position,
+    Paint? paint,
+    int? priority,
+  }) : super(
+          Circle(radius: radius, position: position),
+          paint: paint,
+          priority: priority,
+        );
+}
+
+class RectangleComponent extends ShapeComponent {
+  RectangleComponent(
+    Vector2 size, {
+    Vector2? position,
+    Paint? paint,
+    int? priority,
+  }) : super(
+          Rectangle(size: size, position: position),
+          paint: paint,
+          priority: priority,
+        );
+
+  RectangleComponent.square(
+    double edgeLength, {
```

Comment:
```suggestion
    double size, {
```

Reply:
We have `size` as a `Vector2` everywhere else, so I thought that this would be clearer, but maybe it is not?

Reply:
Yeah, I was a little bit confused by `edgeLength`.  I think I still prefer `size` but maybe, `sideSize` is another option?

Reply:
Let's do size, I think it'll be clear enough since you see the type. :)